Functions/Quantification/xASL_quant_ASL.m

```diff
@@ -163,10 +177,8 @@
     %% 2. Run BASIL quantification
     if bUseBasilQuantification
         % Here we perform FSL quantification
-
-        % #1543 We may want to move the NaN extrapolation here
-
-		[PWI, ATT, ABV, Tex] = xASL_quant_FSL(x.P.Path_PWI4D, x); % #1543 Quick and dirty fix for the NaN extrapolation at the start of xASL_quant_FSL
+		% We pass the path to the image and do all the Image and JSON reading inside the function
```

Comment:
I don't understand. We provide x-struct with general params. And path (NII+JSON) for all specific ones. That seems to be the best flexible solution.

Reply:
What I'm saying, is that in the future, x contains only the settings, and all data-specific parameters (which background suppression, which vendor etc) are only in the JSON.
Currently, we are almost there. We still have x.Q.Background* , x.Q.vendor* , just a few ones.

Reply:
Yes - I fully agree (x will have settings. JSON will have specifics).

So that's why I pass PWI4D_Path (which has the JSON as well) and x-struct. 
So the code should be correct and not require any changes?!

Reply:
Sorry for confusion. Yes, I agree, x.Q.vendor should be removed in the future. 

I meant that apart from this (x.Q.vendor etc) which will be done later, we have everything correct :)